docs/api/batch.md

```diff
@@ -262,6 +262,8 @@ The following status codes can optionally be returned from the API, depending on
 the server implementation.
 
 * 406 - The Accept header needs to be `application/vnd.git-lfs+json`.
+* 413 - The batch API request contained too many objects or the request was
```

Comment:
Would it be worth updating our implementation to return this status code?

Reply:
I assume you're referring to our test server. I've opened git-lfs/lfs-test-server#84 to track implementing that.